contrib/check-config.sh

```diff
@@ -163,7 +163,12 @@ printf -- '- '
 if [ "$(stat -f -c %t /sys/fs/cgroup 2> /dev/null)" = '63677270' ]; then
 	wrap_good 'cgroup hierarchy' 'cgroupv2'
 else
-	cgroupSubsystemDir="$(awk '/[, ](cpu|cpuacct|cpuset|devices|freezer|memory)[, ]/ && $3 == "cgroup" { print $2 }' /proc/mounts | head -n1)"
+	cgroupSubsystemDir=$(grep -e cpu -e cpuacct -e cpuset -e device -e freezer -e memory /proc/mounts | while read -r _ mpoint type _; do
+		if [ "$type" = cgroup ]; then
+			echo "$mpoint"
+			break
+		fi
+	done)
```

Comment:
The biggest change here is that this new `grep` line is looking for `cpu` anywhere in any of the mount lines, which is subtly different from looking for a mount that has `cpu` exactly as a mount flag (and much more likely to have false positives).

Here's what I've got in mind instead:

```suggestion
	cgroupSubsystemDir="$(grep -Em1 '^[^ ]+ [^ ]+ cgroup ([^ ]*,)?(cpu|cpuacct|cpuset|devices|freezer|memory)[, ]' /proc/mounts | cut -d' ' -f2)"
```

Reply:
Yes, I like that! IMO increased readability over the `awk` command plus getting rid of `awk`.

Even though I like `cut` (and have it available in my limited device), I reckon that external `cut` dependency could be replaced shell internal functionality.